src/item_factory.cpp

```diff
@@ -196,7 +196,8 @@ void Item_factory::init()
     iuse_function_list["MYCUS"] = &iuse::mycus;
     iuse_function_list["DOGFOOD"] = &iuse::dogfood;
     iuse_function_list["CATFOOD"] = &iuse::catfood;
-
+    iuse_function_list["CAPTURE_MONSTER_ACT"]=&iuse::capture_monster_act;
+    iuse_function_list["POTION_ACT"]=&iuse::potion_act;
```

Comment:
Well, it was another mechanism to apply healing to, for example, your dog. Seeing as it was also part of the pokemon mod, I'll remove it for the moment as well.